pkg/systemd/init.js

```diff
@@ -200,6 +201,14 @@ define([
                 active_state = load_state + " / " + active_state;
 
             unit.CombinedState = active_state;
+
+            if ( unit.Id.slice(-5) == "timer" ) {
+                unit.is_timer = true;
+                if ( unit.ActiveState == "active" ) {
+                    var timer_unit = systemd_client.proxy('org.freedesktop.systemd1.Timer', unit.path);
+                    wait_valid(timer_unit, add_timer_properties, unit);
```

Comment:
`wait_valid` is only used here - I think it'd be clearer to have the code inline instead of a function.